.gitignore

```diff
@@ -35,3 +35,4 @@ cardigan/
 /server/config/js-assets.json
 
 /server/views/partials/critical.css.njk
+/client/scss/utilities/compiled_variables/
```

Comment:
This worries me in a similar way to the cache break step in the sense it couples this compilation step to the build being successful, with not enforcement on that (make sense?), but also understand the want not to put compiled code in the repo.